models/token_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2016 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+type TokenTestSuite struct {
+	suite.Suite
+}
```

Comment:
@ethantkoenig You're right, we can have only one per package. If that is not enough I'm fine with keeping the suite, but isn't it enough to init the engine once?